chart/values.yaml

```diff
@@ -32,8 +32,8 @@ debug: false
 
 # Extra environment variables passed to the rancher pods.
 # extraEnv:
-# - name: CATTLE_SYSTEM_DEFAULT_REGISTRY
-#   value: "registry.example.com"
+# - name: CUSTOM_ENV_NAME
+#   value: "custom_value"
```

Comment:
Can we use a valid example, e.g. `CATTLE_TLS_MIN_VERSION =1.0`

Reply:
Done, thanks.